### PR TITLE
CHANGELOG for Zebra 1.0.0-alpha.14 

### PR DESCRIPTION
**Pull request by mpguerra**
## Motivation

Update CHANGELOG for Zebra 1.0.0-alpha.14 release

## Review

@ZcashFoundation/zebra-team 

### Reviewer Checklist

  - [ ] Changes are under the correct categories
  - [ ] All significant changes are listed
  - [ ] Descriptions are non-developer friendly

## Follow Up Work

Once release version update PR is merged: 

- [x] review any new merges 
- [x] update CHANGELOG contents if necessary 
- [x] update release date in CHANGELOG

---
This change is [Reviewable](https://reviewable.io/reviews/zcashfoundation/zebra/2538)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,43 @@ All notable changes to Zebra will be documented in this file.
...
+- Pass the finalized state to chain contextual validation (#2503)
+- Calculate incremental note commitment trees (#2407)
+
+#### Documentation
+
+- Add value pools design to book summary (#2520)
+
+#### Testing
+
+- Run `rustfmt` inside nullifier and utxo property tests (#2515)
```
Comment (teor2345) [inline CHANGELOG.md]: `rustfmt` isn't interesting enough to make the changelog
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,43 @@ All notable changes to Zebra will be documented in this file.
...
+- Test consensus-critical Amount deserialization (#2487)
+- Add more details to test coverage increase entry (#2499)
```
Comment (teor2345) [inline CHANGELOG.md]: I'm not sure fixing the previous changelog is that interesting either. But if it is, we should say that we edited the previous changelog.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,43 @@ All notable changes to Zebra will be documented in this file.
...
+- Update to use new GitHub action names in Google Cloud workflows (#2533)
+
+### Changed
+
+- Decrease number of tests in finalized state (#2521)
+- Make nullifier tests faster and consistent with UTXO tests (#2513)
```
Comment (teor2345) [inline CHANGELOG.md]: Oops, these are mine, they weren't very clear
```suggestion
- Decrease the number of randomised test cases in zebra-state (#2521, #2513)
- Make nullifier tests consistent with UTXO tests (#2513)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,43 @@ All notable changes to Zebra will be documented in this file.
...
+- Rename the nullifier test module (#2504)
```
Comment (teor2345) [inline CHANGELOG.md]: Probably doesn't matter that much
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,43 @@ All notable changes to Zebra will be documented in this file.
...
+
+### Fixed
+
+- Clarify indexes in new_ordered_outputs (#2510)
```
Comment (teor2345) [inline CHANGELOG.md]: Oops, this was me again
```suggestion
- Clarify a variable name and check for overflow in new_ordered_outputs (#2510)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,43 @@ All notable changes to Zebra will be documented in this file.
...
+- Reject duplicate Sapling and Orchard nullifiers (#2497)
```
Comment (teor2345) [inline CHANGELOG.md]: "Reject duplicate nullifiers" belongs in `Added`, it's very similar to "Reject UTXO double spends"

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,43 @@ All notable changes to Zebra will be documented in this file.
...
+- Orchard implementation: fixes, refactor, and more test vectors (#2445)
+- clippy: remove a needless borrow (#2501)
```
Comment (teor2345) [inline CHANGELOG.md]: Clippy fixes aren't that interesting
```suggestion
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -4,6 +4,43 @@ All notable changes to Zebra will be documented in this file.
...
+
+- Decrease number of tests in finalized state (#2521)
+- Make nullifier tests faster and consistent with UTXO tests (#2513)
+- Rename the nullifier test module (#2504)
+
+### Fixed
+
+- Clarify indexes in new_ordered_outputs (#2510)
+- Reject duplicate Sapling and Orchard nullifiers (#2497)
+- Orchard implementation: fixes, refactor, and more test vectors (#2445)
```
Comment (teor2345) [inline CHANGELOG.md]: I had to go look up this PR to work out what part of orchard we fixed
```suggestion
- Update Orchard keys, hashes, and note commitments to match Zcash test vectors (#2445)
```

**Review by teor2345 — COMMENTED** [review]
Just some tweaks, nothing major here

File: CHANGELOG.md (hunk 2 above)
Comment (mpguerra) [reply CHANGELOG.md]: ah, I should have looked into the PR to see what it did. FWIW, I don't think it's very interesting either and I have removed it from the PR

File: CHANGELOG.md (hunk 6 above)
Comment (mpguerra) [reply CHANGELOG.md]: Done in 32bd8eec6f3c6923a3affba34fcea69c9a6a66a7

File: CHANGELOG.md [hunk 9]
```diff
@@ -4,6 +4,40 @@ All notable changes to Zebra will be documented in this file.
...
+### Added
+
+- Reject UTXO double spends (#2511)
+- Add a ValueBalance type (#2505)
+- Implement Sum for Amount (#2500)
+- Add an OrderedUtxo type for transparent spend validation (#2502)
+- Pass the finalized state to chain contextual validation (#2503)
+- Calculate incremental note commitment trees (#2407)
+- Reject duplicate Sapling and Orchard nullifiers (#2497)
+- Add `proptest-impl` feature to `zebra-state` crate to help simplify tests(#2529)
```
Comment (teor2345) [inline CHANGELOG.md:20]: Add a missing space
```suggestion
- Add `proptest-impl` feature to `zebra-state` crate to help simplify tests (#2529)
```

**Review by teor2345 — DISMISSED** [review]
The changelog looks good.

The Google cloud build is failing because the branch name in this PR contains `.`, and uppercase letters. (See #2418.)

We can admin-merge this PR as a workaround. Since the failure is caused by the branch name, it should pass after it is merged to `main`.

Edit: Maybe it could be the branch name length or uppercase letters? #2537 passes.

**Comment by dconnolly** [comment]
One more!
https://github.com/ZcashFoundation/zebra/pull/2458

**Comment by teor2345** [comment]
I merged #2542 to fix CI failures

**Comment by teor2345** [comment]
I merged #2486 so I could get started on other work.

**Comment by teor2345** [comment]
Also #2525, sorry about all the changes!

**Comment by mpguerra** [comment]
> Edit: Maybe it could be the branch name length or uppercase letters? #2537 passes.

I feel like I've tried all possible combinations, I thought the branch name length was fine, maybe it is the uppercase.

**Comment by mpguerra** [comment]
going to admin merge once CI ( minus pull-request check) passes and before anything else is merged :P

**Review by mpguerra — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/zcashfoundation/zebra/2538)* status: 0 of 1 files reviewed, 9 unresolved discussions (waiting on @mpguerra and @teor2345)

___
*[CHANGELOG.md, line 26 at r1](https://reviewable.io/reviews/zcashfoundation/zebra/2538#-MfcqhulQZYGDHsPhhtk-r1-26:-MfnHahIBSABRdAZnZiB:b-896fix) ([raw file](https://github.com/zcashfoundation/zebra/blob/93fa535e90b50403fa5076d0cba4578f295c6e95/CHANGELOG.md#L26)):*
<details><summary><i>Previously, teor2345 (teor) wrote…</i></summary><blockquote>

`rustfmt` isn't interesting enough to make the changelog
```suggestion
```
</blockquote></details>

Done.
___
*[CHANGELOG.md, line 28 at r1](https://reviewable.io/reviews/zcashfoundation/zebra/2538#-MfcqhulQZYGDHsPhhtk-r1-28:-MfnHbzp6H-0HqKjUHgn:b-896fix) ([raw file](https://github.com/zcashfoundation/zebra/blob/93fa535e90b50403fa5076d0cba4578f295c6e95/CHANGELOG.md#L28)):*
<details><summary><i>Previously, mpguerra (Pili Guerra) wrote…</i></summary><blockquote>

ah, I should have looked into the PR to see what it did. FWIW, I don't think it's very interesting either and I have removed it from the PR
</blockquote></details>

Done.
___
*[CHANGELOG.md, line 34 at r1](https://reviewable.io/reviews/zcashfoundation/zebra/2538#-MfcqhulQZYGDHsPhhtk-r1-34:-MfnHdED-T7PcQ-lpepF:b-896fix) ([raw file](https://github.com/zcashfoundation/zebra/blob/93fa535e90b50403fa5076d0cba4578f295c6e95/CHANGELOG.md#L34)):*
<details><summary><i>Previously, teor2345 (teor) wrote…</i></summary><blockquote>

Oops, these are mine, they weren't very clear
```suggestion
- Decrease the number of randomised test cases in zebra-state (#2521, #2513)
- Make nullifier tests consistent with UTXO tests (#2513)
```
</blockquote></details>

Done.
___
*[CHANGELOG.md, line 35 at r1](https://reviewable.io/reviews/zcashfoundation/zebra/2538#-MfcqhulQZYGDHsPhhtk-r1-35:-MfnHdwVAlRrW18Z55hS:b-896fix) ([raw file](https://github.com/zcashfoundation/zebra/blob/93fa535e90b50403fa5076d0cba4578f295c6e95/CHANGELOG.md#L35)):*
<details><summary><i>Previously, teor2345 (teor) wrote…</i></summary><blockquote>

Probably doesn't matter that much
```suggestion
```
</blockquote></details>

Done.
___
*[CHANGELOG.md, line 39 at r1](https://reviewable.io/reviews/zcashfoundation/zebra/2538#-MfcqhulQZYGDHsPhhtk-r1-39:-MfnHeT45k2dR8i0qp83:b-896fix) ([raw file](https://github.com/zcashfoundation/zebra/blob/93fa535e90b50403fa5076d0cba4578f295c6e95/CHANGELOG.md#L39)):*
<details><summary><i>Previously, teor2345 (teor) wrote…</i></summary><blockquote>

Oops, this was me again
```suggestion
- Clarify a variable name and check for overflow in new_ordered_outputs (#2510)
```
</blockquote></details>

Done.
___
*[CHANGELOG.md, line 40 at r1](https://reviewable.io/reviews/zcashfoundation/zebra/2538#-MfcqhulQZYGDHsPhhtk-r1-40:-MfnHf4I0IHZn8LgnV7g:b-896fix) ([raw file](https://github.com/zcashfoundation/zebra/blob/93fa535e90b50403fa5076d0cba4578f295c6e95/CHANGELOG.md#L40)):*
<details><summary><i>Previously, mpguerra (Pili Guerra) wrote…</i></summary><blockquote>

Done in 32bd8eec6f3c6923a3affba34fcea69c9a6a66a7
</blockquote></details>

Done.
___
*[CHANGELOG.md, line 41 at r1](https://reviewable.io/reviews/zcashfoundation/zebra/2538#-MfcqhulQZYGDHsPhhtk-r1-41:-MfnHgiwEWgDTigQ5sRW:b-896fix) ([raw file](https://github.com/zcashfoundation/zebra/blob/93fa535e90b50403fa5076d0cba4578f295c6e95/CHANGELOG.md#L41)):*
<details><summary><i>Previously, teor2345 (teor) wrote…</i></summary><blockquote>

I had to go look up this PR to work out what part of orchard we fixed
```suggestion
- Update Orchard keys, hashes, and note commitments to match Zcash test vectors (#2445)
```
</blockquote></details>

Done.
___
*[CHANGELOG.md, line 42 at r1](https://reviewable.io/reviews/zcashfoundation/zebra/2538#-MfcqhulQZYGDHsPhhtk-r1-42:-MfnHfXgCblD4HgWt4KI:b-896fix) ([raw file](https://github.com/zcashfoundation/zebra/blob/93fa535e90b50403fa5076d0cba4578f295c6e95/CHANGELOG.md#L42)):*
<details><summary><i>Previously, teor2345 (teor) wrote…</i></summary><blockquote>

Clippy fixes aren't that interesting
```suggestion
```
</blockquote></details>

Done.
___
*[CHANGELOG.md, line 20 at r5](https://reviewable.io/reviews/zcashfoundation/zebra/2538#-MfcqhulQZYGDHsPhhtk-r5-20:-MfnH_V-7wakL0ZLx1Ig:b-896fix) ([raw file](https://github.com/zcashfoundation/zebra/blob/ec8a7fdfadc53b6b771e362ded734aaa97660697/CHANGELOG.md#L20)):*
<details><summary><i>Previously, teor2345 (teor) wrote…</i></summary><blockquote>

Add a missing space
```suggestion
- Add `proptest-impl` feature to `zebra-state` crate to help simplify tests (#2529)
```
</blockquote></details>

Done.

**Pull request merged**